pythainlp/util/emojiconv.py

```diff
@@ -0,0 +1,1855 @@
+    "🏊‍♂" : "ผู้ชายว่ายน้ำ",
+    "🏋‍♀" : "ผู้หญิงยกน้ำหนัก",
+    "🏋‍♂" : "ผู้ชายยกน้ำหนัก",
+    "🏌‍♀" : "ผู้หญิงตีกอล์ฟ",
+    "🏌‍♂" : "ผู้ชายตีกอล์ฟ",
+    "🚣‍♀" : "ผู้หญิงพายเรือ",
+    "🚣‍♂" : "ผู้ชายพายเรือ",
+    "🚴‍♀" : "ผู้หญิงปั่นจักรยาน",
+    "🚴‍♂" : "ผู้ชายปั่นจักรยาน",
+    "🚵‍♀" : "ผู้หญิงปั่นจักรยานเสือภูเขา",
+    "🚵‍♂" : "ผู้ชายปั่นจักรยานเสือภูเขา",
+    "🤸‍♀" : "ผู้หญิงตีลังกา",
+    "🤸‍♂" : "ผู้ชายตีลังกา",
+    "🤹‍♀" : "ผู้หญิงเล่นจั๊กกลิ้ง",
+    "🤹‍♂" : "ผู้ชายเล่นจั๊กกลิ้ง",
+    "🤼‍♀" : "ผู้หญิงเล่นมวยปล้ำ",
+    "🤼‍♂" : "ผู้ชายเล่นมวยปล้ำ",
+    "🤽‍♀" : "ผู้หญิงเล่นโปโลน้ำ",
+    "🤽‍♂" : "ผู้ชายเล่นโปโลน้ำ",
+    "🤾‍♀" : "ผู้หญิงเล่นแฮนด์บอล",
+    "🤾‍♂" : "ผู้ชายเล่นแฮนด์บอล",
+    "🧘‍♀" : "ผู้หญิงนั่งสมาธิ",
+    "🧘‍♂" : "ผู้ชายนั่งสมาธิ",
+    "👨‍👦" : "ครอบครัว_ผู้ชาย_เด็กชาย",
+    "👨‍👧" : "ครอบครัว_ผู้ชาย_เด็กหญิง",
+    "👩‍👦" : "ครอบครัว_ผู้หญิง_เด็กชาย",
+    "👩‍👧" : "ครอบครัว_ผู้หญิง_เด็กหญิง",
+    "🐈‍⬛" : "แมวดำ",
+    "🐕‍🦺" : "สุนัขบริการ",
+    "🐻‍❄" : "หมีขั้วโลก",
+    "🏳‍⚧" : "ธงคนข้ามเพศ",
+    "🏳‍🌈" : "ธงสีรุ้ง",
+    "🏴‍☠" : "ธงโจรสลัด",
+    "👨‍❤‍👨" : "คู่รัก_ผู้ชาย_ผู้ชาย",
+    "👨‍👦‍👦" : "ครอบครัว_ผู้ชาย_เด็กชาย_เด็กชาย",
+    "👨‍👧‍👦" : "ครอบครัว_ผู้ชาย_เด็กหญิง_เด็กชาย",
+    "👨‍👧‍👧" : "ครอบครัว_ผู้ชาย_เด็กหญิง_เด็กหญิง",
+    "👨‍👨‍👦" : "ครอบครัว_ผู้ชาย_ผู้ชาย_เด็กชาย",
+    "👨‍👨‍👧" : "ครอบครัว_ผู้ชาย_ผู้ชาย_เด็กหญิง",
+    "👨‍👩‍👦" : "ครอบครัว_ผู้ชาย_ผู้หญิง_เด็กชาย",
+    "👨‍👩‍👧" : "ครอบครัว_ผู้ชาย_ผู้หญิง_เด็กหญิง",
+    "👩‍❤‍👨" : "คู่รัก_ผู้หญิง_ผู้ชาย",
+    "👩‍❤‍👩" : "คู่รัก_ผู้หญิง_ผู้หญิง",
+    "👩‍👦‍👦" : "ครอบครัว_ผู้หญิง_เด็กชาย_เด็กชาย",
+    "👩‍👧‍👦" : "ครอบครัว_ผู้หญิง_เด็กหญิง_เด็กชาย",
+    "👩‍👧‍👧" : "ครอบครัว_ผู้หญิง_เด็กหญิง_เด็กหญิง",
+    "👩‍👩‍👦" : "ครอบครัว_ผู้หญิง_ผู้หญิง_เด็กชาย",
+    "👩‍👩‍👧" : "ครอบครัว_ผู้หญิง_ผู้หญิง_เด็กหญิง",
+    "🧑‍🤝‍🧑" : "คนจับมือกัน",
+    "👨‍❤‍💋‍👨" : "จูบ_ผู้ชาย_ผู้ชาย",
+    "👨‍👨‍👦‍👦" : "ครอบครัว_ผู้ชาย_ผู้ชาย_เด็กชาย_เด็กชาย",
+    "👨‍👨‍👧‍👦" : "ครอบครัว_ผู้ชาย_ผู้ชาย_เด็กหญิง_เด็กชาย",
+    "👨‍👨‍👧‍👧" : "ครอบครัว_ผู้ชาย_ผู้ชาย_เด็กหญิง_เด็กหญิง",
+    "👨‍👩‍👦‍👦" : "ครอบครัว_ผู้ชาย_ผู้หญิง_เด็กชาย_เด็กชาย",
+    "👨‍👩‍👧‍👦" : "ครอบครัว_ผู้ชาย_ผู้หญิง_เด็กหญิง_เด็กชาย",
+    "👨‍👩‍👧‍👧" : "ครอบครัว_ผู้ชาย_ผู้หญิง_เด็กหญิง_เด็กหญิง",
+    "👩‍❤‍💋‍👨" : "จูบ_ผู้หญิง_ผู้ชาย",
+    "👩‍❤‍💋‍👩" : "จูบ_ผู้หญิง_ผู้หญิง",
+    "👩‍👩‍👦‍👦" : "ครอบครัว_ผู้หญิง_ผู้หญิง_เด็กชาย_เด็กชาย",
+    "👩‍👩‍👧‍👦" : "ครอบครัว_ผู้หญิง_ผู้หญิง_เด็กหญิง_เด็กชาย",
+    "👩‍👩‍👧‍👧" : "ครอบครัว_ผู้หญิง_ผู้หญิง_เด็กหญิง_เด็กหญิง",
+    "🏴󠁧󠁢󠁥󠁮󠁧󠁿" : "ธง_อังกฤษ",
+    "🏴󠁧󠁢󠁳󠁣󠁴󠁿" : "ธง_สกอตแลนด์",
+    "🏴󠁧󠁢󠁷󠁬󠁳󠁿" : "ธง_เวลส์",
+}
+
+_th_emoji = {v: k for k, v in _emoji_th.items()}
+
+_emojis = sorted(_emoji_th.keys(), key=len, reverse=True)
+_delimiter = ":"
+
+
+def emoji_to_thai(text: str, delimiters=(_delimiter, _delimiter)) -> str:
+    """
+    This function convert emoji to thai meaning
+
+    :param str text_Text with Emoji
+    :return_Text with Emoji being converted to thai meaning
```

Comment:
`:return_Text with Emoji being converted to thai meaning` -> `:return: Text with Emoji being converted to thai meaning`